src/test/java/hw8/test/TestScenario.java

```diff
@@ -0,0 +1,50 @@
+package hw8.test;
+
+import hw8.DataProviderHw8;
+import hw8.beans.ExampleData;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static hw8.JdiSite.*;
+import static hw8.entities.User.ROMAN;
+import static hw8.pages.MetalAndColorsPage.*;
+
+public class TestScenario implements TestInit {
+
+
+    @org.testng.annotations.Test
+    public void loginTest() {
+        open();
+        login(ROMAN);
+        jdiHomePage.checkUserLoggedIn(ROMAN);
+    }
+
+    @Test
+    public void openMetalAndColorsPage() {
+        jdiHomePage.openMetalsAndColorsPage();
+        Assert.assertEquals(getTitle(), titleMetalAndColorsPage);
+    }
+
+    @org.testng.annotations.Test(dataProviderClass = DataProviderHw8.class, dataProvider = "data")
+    public void scenariosTest(ExampleData exampleData) throws InterruptedException {
+
+        radioButtonsOdd.select(exampleData.getSummary().get(0));
```

Comment:
Take a look at the Entity Driving.
It will be better with form.submit(entity)

Reply:
Changed